### ISSUE TITLE
Remove Mozilla Public License

### ISSUE THREAD

**Issue by AirQuick** [xspec/xspec#1786]
https://github.com/xspec/xspec/tree/master/java/com/jenitennison/xslt/tests is under the [Mozilla Public License](https://www.mozilla.org/en-US/MPL/).

The reason I guess is that Jeni started the code from one of the Saxon source code files which originally was contributed by Edwin Glaser. If we remove the remnant of the original code (mostly comments, I guess) and get Jeni's approval, we're probably allowed to change its license to MIT.

(Also we may want to change its namespace from `com.jenitennison.xslt.tests` to `io.xspec.xspec.saxon.something`.)

**Comment by AirQuick** [comment]
> the remnant of the original code (mostly comments, I guess)

Indeed, comparing these files, the original Mozilla Public License portion appears to be, essentially, only comments.
- `net/sf/saxon/trace/TraceListener.java` in [Saxon 9.2.1.2](https://github.com/Saxonica/Saxon-Archive/blob/main/Saxon-HE/9/9.2/saxon9-2-1-2source.zip)
- Jeni's [initial commit](https://github.com/xspec/xspec/commit/54f6e15de57cf90d8180cf1896f159d9948d61ed#diff-d35dbd6d6158b435c4d2a4f317b6dadf0c1172773ceda7045063dc79052c9c84)

**Comment by galtm** [comment]
@JeniT : do you have any comments on this issue?

**Comment by JeniT** [comment]
I don't have any objections to changing the licence or the namespace :)

**Comment by galtm** [comment]
@JeniT : Thanks for your reply.

@AirQuick : Did you have in mind something like the following?

1. In the Java source file, remove the comments that Jeni didn't change between the Saxon 9.2.1.2 `TraceListener.java` file and her initial commit of the `XSLTCoverageTraceListener.java` file.
2. Change the set of comments at the bottom of the file to something that gives credit without asserting the Mozilla license. Perhaps something like the block of code below
3. In README.md at the top of this repo, remove the sentence that mentions the Mozilla license.

```
// This file is part of the XSpec repository, github.com/xspec/xspec.
//
// Development credits:
// - Methods implemented by Jeni Tennison
// - Extended for Saxon 9.7 by Sandro Cirulli, github.com/cirulls
// - Extended for Saxon 12.4 by AirQuick, github.com/AirQuick
//
// Portions created by Jeni Tennison are Copyright (C) Jeni Tennison.
// All Rights Reserved.
//
// This file was inspired by net/sf/saxon/trace/TraceListener.java
// in Saxon 9.2.1.2. That file was developed by Edwin Glaser and
// heavily modified by Michael Kay.
```

Cc: @xspec/xspec to keep others informed and in case anyone else has advice or opinions.
